C-git-commands.asc

```diff
@@ -568,6 +568,13 @@ In <<ch07-git-tools#_removing_file_every_commit>> we explain the command and exp
 
 In <<ch09-git-and-other-systems#_git_p4>> and <<ch09-git-and-other-systems#_git_tfs>> we use it to fix up imported external repositories.
 
+[CAUTION]
+====
+`git filter-branch` is no longer the recommended way to rewrite history.
+Instead use the Python script `git filter-repo`, this script is safer and more performant.
+This script is not included with git, you will need to download and install the script yourself.
+Read the documentation for `git filter-repo` [here](https://github.com/newren/git-filter-repo/).
+====
```

Comment:
Let's drop this diff. This section is only listing places where a reader can find mentions of specific commands, kind of like an index, and it's not the right place for this kind of warning.